<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesDlg</class>
 <widget class="QDialog" name="PropertiesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="dht">
     <property name="toolTip">
      <string>DHT is a distributed database which can be used to find more peers for a torrent.</string>
     </property>
     <property name="text">
      <string>Use DHT to find more peers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pex">
     <property name="toolTip">
      <string>Peer exchange results in more peers being found, by exchanging information about peers with other peers.</string>
     </property>
     <property name="text">
      <string>Use peer exchange to find more peers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="superseeding">
     <property name="toolTip">
      <string>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Superseeding mode, is a special seeding mode which allows you to achieve much higher seeding efficiencies. This results in less bandwidth used to spread a torrent.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note: &lt;/span&gt;This should only be used if the swarm has not fully been established. Once there are multiple active seeders, this mode becomes useless.&lt;/p&gt;</string>
     </property>
     <property name="text">
      <string>Use superseeding mode for seeding</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="move_on_completion_enabled">
     <property name="toolTip">
      <string>Move the files of the torrent to a different directory when it has finished downloading.</string>
     </property>
     <property name="text">
      <string>Move when completed to:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="move_on_completion_url">
     <property name="toolTip">
      <string>Location to move the files to.</string>
     </property>
     <property name="mode">
      <set>KFile::Directory|KFile::ExistingOnly|KFile::LocalOnly</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
